lib/workers/repository/process/lookup/index.ts

```diff
@@ -345,6 +349,8 @@ export async function lookupUpdates(
     logger.debug(`Dependency ${depName} has unsupported value ${currentValue}`);
     if (!config.pinDigests && !config.currentDigest) {
       res.skipReason = 'unsupported-value';
+    } else {
+      delete res.skipReason;
```

Comment:
In which cases does this become useful?

Reply:
test/workers/repository/process/lookup/index.spec.ts:1224

Reply:
https://github.com/renovatebot/renovate/blob/master/test/workers/repository/process/lookup/index.spec.ts#L1223